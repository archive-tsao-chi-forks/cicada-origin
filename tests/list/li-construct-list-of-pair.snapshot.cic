@show
@the List((String * String))
@li @the (String * String) cons("cooked", "cooked")
@li @the (String * String) cons("eaten", "eaten")
@li @the (String * String) cons("tried-cleaning", "tried-cleaning")
@li @the (String * String) cons("understood", "understood")
@li @the (String * String) cons("slept", "slept")
@nil
