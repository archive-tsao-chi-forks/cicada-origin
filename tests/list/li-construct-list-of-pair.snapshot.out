@the List((_: String * String)) li(cons("slept", "slept"), li(cons("understood", "understood"), li(cons("tried-cleaning", "tried-cleaning"), li(cons("eaten", "eaten"), li(cons("cooked", "cooked"), nil)))))

